app/views/admin/shared/v3/_pagy.html.haml

```diff
@@ -2,9 +2,6 @@
   - if pagy.prev
     %a.page.prev{ href: "#", id: "pagy-prev", "data-reflex": reflex, "data-perPage": pagy.items, "data-page": pagy.prev || 1, "aria-label": "previous"}
       %i.icon-chevron-left
-  - else
-    %span.page.prev.disabled
```

Comment:
I guess the `.disabled` css rules can also be removed now.

Reply:
Oh, I guess so ... Thanks!